state/protocol/protocol_state/kvstore/models.go

```diff
@@ -114,6 +118,11 @@ func (model *Modelv0) SetEpochStateID(id flow.Identifier) {
 	model.EpochStateID = id
 }
 
+// GetEpochExtensionViewCount returns the length of the epoch extension in views.
```

Comment:
if `EpochExtensionViewCount` is in the Protocol State that means its value can possibly change at runtime.  So we can't just say that epoch extensions have these numbers of views (which I would interpret as _all_ epoch extensions are having these number of views) 

```suggestion
// GetEpochExtensionViewCount returns the number of views for a hypothetical epoch extension. Note 
// that this value can change at runtime (through a service event). When a new extension is added, 
// the view count is used right at this point in the protocol state's evolution. In other words, 
// different extensions can have different view counts.
```